asreview/webapp/authentication/remote_user_handler.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2019-2022 The ASReview Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+class RemoteUserHandler:
+    default_headers = {
+        "USER_IDENTIFIER_HEADER": "REMOTE_USER",
+        "USER_NAME_HEADER": False,
+        "USER_EMAIL_HEADER": False,
+        "USER_AFFILIATION_HEADER": False,
+        "DEFAULT_EMAIL_DOMAIN": "localhost",
+        "DEFAULT_AFFILIATION": "",
```

Comment:
Maybe make the default value `None`. Now a missing identifier will be returned as `None` but a missing affiliations is returned as `""`.